spec/symengine_spec.rb

```diff
@@ -18,5 +18,16 @@
     it { is_expected.to be_a SymEngine::Rational }
     its(:to_s) { is_expected.to eq '1/3' }
   end
+  
+  describe 'parse' do
+    subject { SymEngine::parse('123 + 321') }
```

Comment:
I would like to add a few more to this list
- `5+-3`
- `5--3`

*All Without spaces*
And
- `5++3` or something like this to raise an error.